src/base/rss/rss_session.cpp

```diff
@@ -31,14 +31,12 @@
 #include "rss_session.h"
 
 #include <QDebug>
-#include <QDir>
 #include <QJsonDocument>
 #include <QJsonObject>
 #include <QJsonValue>
 #include <QSaveFile>
 #include <QString>
 #include <QThread>
-#include <QVariantHash>
```

Comment:
I'm wary of PRs like this. It might not be so obvious to change it like that. On what principle do you do it? For example, why did you delete this include?

Reply:
There is no QVariant, QVariantHash in the file and it compiles?

Reply:
>There is no QVariant, QVariantHash

Are you sure? Or were you just searching for QVariant and QVariantHash occurrences? It isn't enough in common case. They can come from other includes.
>and it compiles

This headers can be included indirectly (via other headers).

Okay, whatever. If we systematically replace includes with forward declarations in our headers, then it calms down in the end.